docs/install.md

```diff
@@ -270,7 +270,7 @@ To customize the behavior of the Pipelines Controller, modify the ConfigMap `fea
 
 - `disable-affinity-assistant` - set this flag to disable the [Affinity Assistant](./workspaces.md#affinity-assistant-and-specifying-workspace-order-in-a-pipeline)
   that is used to provide Node Affinity for `TaskRun` pods that share workspace volume. 
-  The Affinity Assistant pods may be incompatible with NodeSelector and other affinity rules
+  The Affinity Assistant is incompatible with other affinity rules
   configured for `TaskRun` pods.
```

Comment:
would it make sense to document that specifically nodeselector and tolerations will be propogated from the podtemplate?

it feels like this is a significant enough feature that we might want to add a .md doc just to explain the affinity assistant and how to configure it, what do you think?

Reply:
Yes, we have some [documentation, about what the Affinity Assistant](https://github.com/tektoncd/pipeline/blob/master/docs/workspaces.md#affinity-assistant-and-specifying-workspace-order-in-a-pipeline) **is**, and I can add docs about these two propagated fields there.

The only configuration the user can do currently, is to disable it. But we can also write more extensive docs about it. I can have a look about more extensive docs tomorrow.

Reply:
Thanks for the review, I added info about field propagation to the workspaces.md doc.